assignments/3. type conversion/index.md

```diff
@@ -1,36 +1,36 @@
 1. 🎖 What's the outpit of the code below
 ```js
-typeof "Joe";
-typeof 4;
-typeof NaN;
-typeof false;
-typeof function () {};
-var phone = 8983700;
-typeof phone;
-typeof null;
+typeof "Joe"; //string
+typeof 4; //number
+typeof NaN; //number
+typeof false; //boolean
+typeof function () {}; //function
+var phone = 8983700; 
+typeof phone; //number
+typeof null; //object
 ```
 
 2. 🎖 Output of the code below
 ```js
 // Convert num into string
-var num = 45;
-String(num);
-String(321);
-String(300 + 23);
-String(false);
-String(true);
-Number("3.18");
-Number(" ");
-Number("");
-Number("22 44");
-Number(false);
-Number(true);
+var num = 45; 
+String(num); //"45"
+String(321); //"321"
+String(300 + 23); //"323"
+String(false); //"False"
```

Comment:
not `"False"` it should be `"false"`. JavaScript is a case sensitive language so be specific.